modules/@apostrophecms/doc-type/ui/apos/components/AposDocEditor.vue

```diff
@@ -3,6 +3,7 @@
     class="apos-doc-editor"
     :modal="modal"
     :modal-title="modalTitle"
+    :modal-data="modalData"
```

Comment:
Prop comes from `TheAposModals.vue` we need it here to get the current modal locale.